### PR TITLE
feat: Made the website look actually nice

### PR DESCRIPTION
**Pull request by odysseuskir**

### CODE REVIEW

File: src/public/index.html [hunk 1]
```diff
@@ -5,33 +5,56 @@
...
+        main > div {
+            max-width: 600px;
+            width: 100%;
+            text-align: center;
+        }
+
+        main > div > p {
+          margin-top: 20px;
+        }
+    </style>
```
Comment (GalaxyGamingBoy) [inline src/public/index.html:34]: Use TailwindCSS

**Review by GalaxyGamingBoy — CHANGES_REQUESTED** [review]
Use TailwindCSS

File: src/public/index.html (hunk 1 above)
Comment (odysseuskir) [reply src/public/index.html:34]: No, you use Tailwind

**Comment by GalaxyGamingBoy** [comment]
Closing PR#2 till the issue is resolved, people use different architectures for a reason

**Comment by odysseuskir** [comment]
Well sorry, just wanted to make it nice. Besides, it's nothing significant, just a landing page.

**Pull request merged**